SpatialGDK/Source/SpatialGDK/Private/SpatialGDKModule.cpp

```diff
@@ -13,7 +13,14 @@ DEFINE_LOG_CATEGORY(LogSpatialGDKModule);
 
 IMPLEMENT_MODULE(FSpatialGDKModule, SpatialGDK)
 
-void FSpatialGDKModule::StartupModule() {}
+void FSpatialGDKModule::StartupModule()
+{
+	if (FString(FCommandLine::Get()).Contains(TEXT("-SpatialTest")))
+	{
+		FModuleManager::Get().LoadModule("SpatialGDKServices");
```

Comment:
Why do we need this?

Reply:
maybe add a comment here

Reply:
Yeah sorry should have added a comment here, basically to run the current GDK’s test gyms we have to include the SpatialFunctionalTests and the SpatialGDKServices modules, which have a Target type of Editor. Newly-spawned processes still have the Editor target type, but Unreal prevents loading a module with an Editor target type from a process that is not the editor itself. Therefore, we need to somehow force load the 2 modules mentioned above and we thought of doing this here as the least intrusive way of forcing this.  

Do you have another suggestion on how to achieve this, cause even in this form (I think the 3rd try of doing this 😂) it is still a bit ugly?

Reply:
(by the way, if not resolvable another way, what I meant by the comment is a comment in code, so other people can also question this and get an answer)

Reply:
Oh yeah, I was talking about adding a comment in the code itself but I also wanted to add this here so that people reading this still have some context :)